include/sha256_ni.h

```diff
@@ -12,8 +12,6 @@
 
 void sha256_ni(uint8_t* out, const uint8_t* in, const uint64_t length);
 
-void recursive_sha256_ni(uint8_t* hash, const uint64_t num_iters);
```

Comment:
You can leave it here and just include the same file in your *.cpp

Reply:
Fixed & commited.